test/helpers/browser.js

```diff
@@ -99,7 +99,6 @@ export const setup = () => {
 	const initSingleInstance = async () => {
 		const page = await browser.newPage();
 		await page.goto(C.singleInstanceUrl());
-		await page.evaluate(() => typeof window.singleInstanceApi !== 'undefined');
```

Comment:
Can you explain why this was removed?

Reply:
Short answer: It does nothing except cause unnecessary test failures

Long answer: Few reasons
1) What evaluate does is run some code in an injected script then return the result. This did a boolean check and then...do nothing with the result.
2) The other init functions called waitForFunction instead of evaluate, so I figured it was a typo. But then swapping it out caused test failures to happen all the time.
3) The test that failed because of that line is `singleInstance - should redirect to busy.html when the instance is already taken`. Basically injecting code (using evaluate) caused failures because it was redirecting, leading to a race condition. waitForFunction similarly failed because it was redirecting, so that if condition was no longer true (and its not supposed to be true if it redirects).

There might be a better alternative than removing (a third option), but I'm not familiar enough to make that call.

Reply:
Whoops, you're right. That code does nothing. I wonder if I meant for that to be a recurring check that waited for `window.singleInstanceApi` to exist before resolving the promise.

Reply:
Probably. It definitely looked like it was meant to be like the waitForFunctions above. I was going to make it consistent but then that redirect caused hella craziness. Felt sad not to see an assurance that it navigated correctly, but page.goto does a waitUntil: load anyways so its most likely fine.

Funny enough, I nearly commited a bunch of `, { waitUntil: 'load' }` before I realized that load is the default. Puppeteer is so much better than selenium.